pymc3/backends/tracetab.py

```diff
@@ -21,16 +21,13 @@ def trace_to_dataframe(trace, chains=None, flat_names=None):
     """
     var_shapes = trace._straces[0].var_shapes
     if flat_names is None:
-        flat_names = {v: create_flat_names(v, shape)
-                      for v, shape in var_shapes.items()
-                      if not v.endswith('_')}
+        flat_names = {v: create_flat_names(v, shape) for v, shape in var_shapes.items()}
```

Comment:
Ah, you mentioned that. Perhaps we can add a `kwarg` like `hide_transformed_vars`?